server/src/domain/asset/dto/asset.dto.ts

```diff
@@ -10,6 +22,10 @@ export enum AssetOrder {
   DESC = 'desc',
 }
 
+const hasGPS = (o: { latitude: undefined; longitude: undefined }) =>
+  o.latitude !== undefined || o.longitude !== undefined;
+const ValidateGPS = () => ValidateIf(hasGPS);
+
```

Comment:
Can we use `@isLatitude()` and `@isLongitude()` instead of building our own?

Reply:
We are not using or own, is to check if both of the parmaeter are given (you can't give longitude without latitude)

Reply:
Oh sorry, my fault!